src/plugins/lanplus/lanplus.h

```diff
@@ -78,6 +78,25 @@
 
 #define IPMI_CRYPT_AES_CBC_128_BLOCK_SIZE 0x10
 
+#define IPMI_LANPLUS_CIPHER_SUITE_0 0
+#define IPMI_LANPLUS_CIPHER_SUITE_1 1
+#define IPMI_LANPLUS_CIPHER_SUITE_2 2
+#define IPMI_LANPLUS_CIPHER_SUITE_3 3
+#define IPMI_LANPLUS_CIPHER_SUITE_4 4
+#define IPMI_LANPLUS_CIPHER_SUITE_5 5
+#define IPMI_LANPLUS_CIPHER_SUITE_6 6
+#define IPMI_LANPLUS_CIPHER_SUITE_7 7
+#define IPMI_LANPLUS_CIPHER_SUITE_8 8
+#define IPMI_LANPLUS_CIPHER_SUITE_9 9
+#define IPMI_LANPLUS_CIPHER_SUITE_10 10
+#define IPMI_LANPLUS_CIPHER_SUITE_11 11
+#define IPMI_LANPLUS_CIPHER_SUITE_12 12
+#define IPMI_LANPLUS_CIPHER_SUITE_13 13
+#define IPMI_LANPLUS_CIPHER_SUITE_14 14
+#define IPMI_LANPLUS_CIPHER_SUITE_15 15
+#define IPMI_LANPLUS_CIPHER_SUITE_16 16
+#define IPMI_LANPLUS_CIPHER_SUITE_17 17
```

Comment:
make these an enum type instead, please. enums are symbols that can be seen/resolved during gdb session, unlike macros. change the associated variables/struct fields to that type too, please.